plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSink.java

```diff
@@ -128,6 +131,7 @@ public IcebergPageSink(
         this.jsonCodec = requireNonNull(jsonCodec, "jsonCodec is null");
         this.session = requireNonNull(session, "session is null");
         this.fileFormat = requireNonNull(fileFormat, "fileFormat is null");
+        this.metricsConfig = MetricsConfig.fromProperties(requireNonNull(storageProperties, "metricsConfig is null"));
```

Comment:
error message does not match. 

Btw, I think it makes more sense to pass in the `MetricsConfig` to the constructor instead of `storageProperties`, because file format is also derived from `storageProperties` and we are passing it in separately.

Reply:
@jackye1995 Sorry you meant passing in the `MetricsConfig` to the `pagesink` or `IcebergWritableTableHandle` constructor? 

The file format is derived from the property map and then passed to the writable table handle as a separate component, which is how we passed in`MetricsConfig` initially.

Reply:
to the page sink. But I don't have a strong opinion on this, there are many duplicated information passed across the codebase as of today. We can just fix the error message first.

Reply:
> @liqinrae can you please check the Ci build results?

@findepi Yes seems like there is new changes from other PRs. Will merge them and fix the CI build.